distributions/raspijamulus.sh

```diff
@@ -22,10 +23,10 @@ fi
 if [ -d "${OPUS}" ]; then
   echo "The Opus directory is present, we assume it is compiled and ready to use. If not, delete the opus directory and call this script again."
 else
-  wget https://archive.mozilla.org/pub/opus/${OPUS}.tar.gz
-  tar -xzf ${OPUS}.tar.gz
-  rm ${OPUS}.tar.gz
-  cd ${OPUS}
+  wget https://archive.mozilla.org/pub/opus/"${OPUS}.tar.gz"
+  tar -xzf "${OPUS}.tar.gz"
+  rm "${OPUS}.tar.gz"
+  cd "${OPUS}" || exit
```

Comment:
If exiting due to an error, use a non-zero exit status, e.g. `exit 1`. Similarly below on line 66. Then the caller can tell the script failed, if necessary.

Reply:
Done.